cfgov/unprocessed/css/organisms/read-more.less

```diff
@@ -1,102 +1,3 @@
-          </div>
-          <button class="o-expandable_header o-expandable_target"
-                  title="Expand content">
-              <span class="o-expandable_link">
-                  <span class="o-expandable_cue o-expandable_cue-open">
-                      Read full answer
-                      {{ svg_icon('plus-round') }}
-                  </span>
-                  <span class="o-expandable_cue o-expandable_cue-close">
-                      Hide full answer
-                      {{ svg_icon('minus-round') }}
-                  </span>
-              </span>
-          </button>
-      </div>
-    codenotes:
-      - |
-        Pattern structure
-        -----------------
-        .o-expandable
-          .o-expandable_content
-          .o-expandable_header.o-expandable_target
-            .o-expandable_link
-              .o-expandable_cue.o-expandable_cue-open
-              .o-expandable_cue.o-expandable_cue-close
-
-    notes:
-      - "One-way expandable. Displays an approximately 4-line
-         preview of content on mobile screen sizes only. "
-  - name: .o-expandable__read-more-mobile (modifier)
-  tags:
-  - cf-organisms
-*/
-
-.o-expandable__read-more-mobile {
-    .o-expandable_target__collapsed,
-    .o-expandable_content__collapsed:after {
-        display: none;
-    }
-
-    .o-expandable_content {
-        // These two !important values override basic expandable styling,
-        // because these don't function like expandables on sm+ screens.
-        display: block !important;
-        max-height: 100% !important;
-    }
-
-    @media only screen and (max-width: @bp-xs-max) {
-        .o-expandable_target__collapsed {
-            display: block;
-
-            .o-expandable_link {
-                display: block;
-                padding-top: 15px;
-                padding-bottom: 15px;
-                border: dotted @link-text;
-                border-width: 1px 0;
-                text-align: center;
-            }
-        }
-
-        .o-expandable_target__expanded {
-            display: none;
-        }
-
-        .o-expandable_content__collapsed {
-            // shows approx. 4 lines of content
-            // !important needed to override the default o-expandable_content__collapsed
-            // max-height
-            max-height:  unit( @base-line-height-px * 4 / @base-font-size-px, em ) !important;
-            overflow: hidden;
-            position: relative;
-
-            &:after {
-                // fades content out over approx. 2 lines
-                display: block;
-                height: unit( @base-line-height-px * 2 / @base-font-size-px, em );
-                margin: 0;
-                position: absolute;
-                bottom: 0;
-                left: 0;
-                right: 0;
-                background: linear-gradient(to bottom, rgba(255,255,255,0) 0%,rgba(255,255,255,1) 100% );
-                content: '';
-            }
-        }
-    }
-}
-
```

Comment:
All the stuff below here is still in use somewhere?

Reply:
Yeah it's the "Show full list of relevant parties" toggle [here](https://www.consumerfinance.gov/data-research/prepaid-accounts/search-agreements/detail/48439/).

Reply:
Why doesn't that use a regular expandable?

Reply:
It has some customization. The target (the "Show full list of relevant parties") has margin and other [minor tweaks added](https://github.com/cfpb/consumerfinance.gov/blob/86c6533dc9d41b1b43cf66041c00d0c80af2877a/cfgov/unprocessed/css/organisms/read-more.less#L55-L57). Additionally, it appears it was [intended to show a summary](https://github.com/cfpb/consumerfinance.gov/blob/86c6533dc9d41b1b43cf66041c00d0c80af2877a/cfgov/unprocessed/css/organisms/read-more.less#L49), which is not showing because `o-expandable_content` has `u-hidden` set. It should either use an expandable or the summary component in this PR I think, but I'd address it in its own PR.

Reply:
I was more asking about why those design decisions were made. I'm fine with it not holding up this PR, but let's keep the ball rolling. @virginiacc, looks like you added this code. Do you remember who was on the design side of that?

Reply:
https://github.cfpb.gov/CFGOV/platform/issues/3980